Assets/_BForBoss/_Core/Scripts/Environment/WallRunning/WallRunSO.cs

```diff
@@ -0,0 +1,66 @@
+using Sirenix.OdinInspector;
+using UnityEngine;
+
+namespace BForBoss
+{
+    [CreateAssetMenu(fileName = "Wall Run Data", menuName = "PerigonGames/WallRunData", order = 2)]
+    public class WallRunSO : ScriptableObject
+    {
+        [FoldoutGroup("Wall Run Movement properties")]
+        [SerializeField] 
+        private float _speedMultiplier = 1f;
+        [FoldoutGroup("Wall Run Movement properties")]
+        [SerializeField]
+        private float _maxWallRunAcceleration = 20f;
+        [FoldoutGroup("Wall Run Movement properties")]
+        [SerializeField]
+        private float _wallGravityDownForce;
+
+        [FoldoutGroup("Timers")]
+        [SerializeField, Tooltip("Gravity won't apply until after this many seconds")]
+        private float _gravityTimerDuration = 1f;
+        [FoldoutGroup("Timers")]
+        [SerializeField, Tooltip("Wall runs on the same wall are only allowed after this long")]
+        private float _wallResetTimer = 2f;
+
+        [FoldoutGroup("Wall Run jump properties")]
+        [SerializeField]
+        private float _wallBounciness = 6f;
+        [FoldoutGroup("Wall Run jump properties")]
+        [SerializeField]
+        private float _jumpHeightMultiplier = 1f;
+        [FoldoutGroup("Wall Run jump properties")]
+        [SerializeField, Range(0f, 2f)]
+        private float _jumpForwardVelocityMultiplier = .75f;
+
+        [FoldoutGroup("Camera Settings")]
+        [SerializeField]
+        private float _maxCameraAngleRoll = 30f;
+        [FoldoutGroup("Camera Settings")]
+        [SerializeField]
+        private float _cameraRotateDuration = 1f;
+        [FoldoutGroup("Camera Settings")]
+        [SerializeField] 
+        private float _lookAlongWallRotationSpeed = 3f;
```

Comment:
I think this field and `_minLookAlongWallStabilizationAngle` should have tooltips as well

Reply:
Updated